src/azure-cli/azure/cli/command_modules/servicebus/tests/latest/test_servicebus_alias_commands.py

```diff
@@ -160,7 +160,7 @@ def test_sb_alias(self, resource_group):
         time.sleep(30)
 
         # Delete Namespace - primary
-        self.cmd('servicebus namespace delete --resource-group {rg} --name {namespacenameprimary}')
+        self.cmd('servicebus namespace delete --resource-group {rg} --name {namespacenameprimary} --yes')
 
         # Delete Namespace - secondary
-        self.cmd('servicebus namespace delete --resource-group {rg} --name {namespacenamesecondary}')
+        self.cmd('servicebus namespace delete --resource-group {rg} --name {namespacenamesecondary} --yes')
```

Comment:
If there's no confirmation before, then it's breaking change to add double confirmation. The existing automation script could be blocked

Reply:
Removed the confirmation message.